aspnetcore/web-api/jsonpatch.md

```diff
@@ -14,6 +14,29 @@ By [Tom Dykstra](https://github.com/tdykstra)
 
 This article explains how to handle JSON Patch requests in an ASP.NET Core web API.
 
+::: moniker range=">= aspnetcore-3.0"
+
+## Package installation
+
+Support for JsonPatch is enabled using the `Microsoft.AspNetCore.Mvc.NewtonsoftJson` package. To enable this feature, apps must:
+
+* Install the [Microsoft.AspNetCore.Mvc.NewtonsoftJson](https://www.nuget.org/packages/Microsoft.AspNetCore.Mvc.NewtonsoftJson/) NuGet package.
+* Update the project's `Startup.ConfigureServices` method to include a call to `AddRazorRuntimeCompilation`:
```

Comment:
Should this be `AddNewtonsoftJson` instead of `AddRazorRuntimeCompilation`?

Reply:
Oops